testhelp/.gitignore

```diff
@@ -0,0 +1 @@
+/fatal
```

Comment:
what's contained in the 'fatal' directory?

Reply:
`make test` generates a file named `fatal` under that path.